ecosystem/sep-0041.md

```diff
@@ -0,0 +1,239 @@
+token interfaces defined in [CAP-46-6].
+
+## Motivation
+
+A fungible asset is a fundamental concept on blockchains. Fungible assets can
+conceptually be divided into two pieces: standard functionality enabling push
+and pull transfers, and varying functionality around asset administration. Most
+blockchain ecosystems have very little innovation in the space of fungible
+assets, with developers often relying on open source implementations such as
+OpenZeppelin.
+
+[CAP-46-6] defines a Stellar Asset contract that implements a combination of
+that standard functionality, as well as functions to support behaviors that are
+specialized to Stellar assets.
+
+Tokens could implement the interface that the Stellar Asset contract defines,
+but it contains functions that support behaviors that are specialized to Stellar
+assets.
+
+An interface is needed that is less opionated than the interface of the Stellar
+Asset contract. Tokens that implement this interface would support at least the
+standard functionality of tokens, without needing to support the specialized
+behaviors of Stellar Assets.
+
+An interface is needed that is a subset of the Stellar Asset contract, such that
+the Stellar Asset contract would implement the interface. Tokens that implement
+the interface and the Stellar Asset contract would be usable with contracts that
+support the standard interface, and would be largely indistinguishable.
+
+[CAP-46-6]: ../core/CAP-0046-06.md
+
+## Specification
+
+### Interface
+
+```rust
+pub trait TokenInterface {
+    /// Returns the allowance for `spender` to transfer from `from`.
+    ///
+    /// # Arguments
+    ///
+    /// - `from` - The address holding the balance of tokens to be drawn from.
+    /// - `spender` - The address spending the tokens held by `from`.
+    fn allowance(env: Env, from: Address, spender: Address) -> i128;
+
+    /// Set the allowance by `amount` for `spender` to transfer/burn from
+    /// `from`.
+    ///
+    /// # Arguments
+    ///
+    /// - `from` - The address holding the balance of tokens to be drawn from.
+    /// - `spender` - The address being authorized to spend the tokens held by
+    /// `from`.
+    /// - `amount` - The tokens to be made available to `spender`.
+    /// - `expiration_ledger` - The ledger number where this allowance expires.
+    /// Cannot be less than the current ledger number unless the amount is being
+    /// set to 0.  An expired entry (where expiration_ledger < the current
+    /// ledger number) should be treated as a 0 amount allowance.
+    ///
+    /// # Events
+    ///
+    /// Emits an event with topics `["approve", from: Address,
+    /// spender: Address], data = [amount: i128, expiration_ledger: u32]`
+    ///
+    /// Emits an event with:
+    /// - topics - `["approve", from: Address, spender: Address]`
+    /// - data - `[amount: i128, expiration_ledger: u32]`
+    fn approve(env: Env, from: Address, spender: Address, amount: i128, expiration_ledger: u32);
```

Comment:
`expiration_ledger` is being renamed to `live_until_ledger`.

Reply:
```suggestion
    fn approve(env: Env, from: Address, spender: Address, amount: i128, live_until_ledger: u32);
```